doc/api/api_changes/2017-02-26-MB_intersects_bbox.rst

```diff
@@ -0,0 +1,21 @@
+Path.intersects_bbox always treats the bounding box as filled
+`````````````````````````````````````````````````````````````
+
+Previously, when ``Path.intersects_bbox`` was called with ``filled`` set to
+``False``, it would treat both the path and the bounding box as unfilled. This
+behavior was not well documented and it is usually not the desired behavior,
+since bounding boxes are used to represent more complex shapes located inside
+the bounding box. This behavior has now been changed: when ``filled`` is
+``False``, the path will be treated as unfilled, but the bounding box is still
+treated as filled. The old behavior was arguably an implementation bug.
+
+When ``Path.intersects_bbox`` is called with ``filled`` set to ``True``
+(the default value), there is no change in behavior. For those rare cases where
+``Path.intersects_bbox`` was called with ``filled`` set to ``False`` and where
+the old behavior is actually desired, the suggested workaround is to call
+``Path.intersects_path`` with a rectangle as the path:
```

Comment:
I think you need to end the paragraph with two colons (`::`) so that what follows is treated as a code block.
You can just `commit --amend` and force push.